client/me/purchases/marketplace-subscriptions-dialog/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { Dialog } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { Fragment } from 'react';
+import FormSectionHeading from 'calypso/components/forms/form-section-heading';
+
+interface MarketPlaceSubscriptionsDialogProps {
+	planName: string;
+	closeDialog: () => void;
+	removePlan: () => void;
+	isDialogVisible: boolean;
+	isRemoving: boolean;
+	activeSubscriptions: Array< { id: number; productName: string } >;
+}
+
+export const MarketPlaceSubscriptionsDialog = ( {
+	planName,
+	closeDialog,
+	removePlan,
+	isDialogVisible,
+	activeSubscriptions,
+}: MarketPlaceSubscriptionsDialogProps ): JSX.Element => {
+	const translate = useTranslate();
+
+	const buttons = [
+		{
+			action: 'cancel',
+			label: translate( 'Cancel' ),
+			onClick: closeDialog,
+		},
+		{
+			isPrimary: true,
+			action: 'removePlanAndAllSubscriptions',
+			label: translate( 'Remove Plan & All Subscriptions', {
+				comment:
+					'This button removes the active plan and all active Marketplace subscriptions on the site',
+			} ),
+			onClick: removePlan,
+		},
+	];
+
+	return (
+		<Dialog
+			buttons={ buttons }
+			className="marketplace-subscriptions-dialog"
+			isVisible={ isDialogVisible }
+			onClose={ closeDialog }
+		>
+			<Fragment>
+				<FormSectionHeading>
+					{ translate( 'Remove %(plan)s', {
+						args: { plan: planName },
+					} ) }
+				</FormSectionHeading>
+				<p>
+					{ translate(
+						'When you downgrade your plan, the following subscription will remain active:',
```

Comment:
```suggestion
						'The following subscriptions depend on your plan:',
```

<img width="1029" alt="SS 2022-01-12 at 20 39 17" src="https://user-images.githubusercontent.com/12430020/149201732-abcaa41c-09ae-4cde-831c-b1fbd625b6c1.png">

Reply:
Good point, sound much more logical like this 😃